packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -170,6 +187,13 @@ export interface FileSystemProps {
    * @default RemovalPolicy.RETAIN
    */
   readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * Enum to turn automatic backups on or off for the file system.
```

Comment:
the documentation leads me to think this maybe should be a boolean if there are only 2 valid values